spring-integration-cloudevents/src/main/java/org/springframework/integration/cloudevents/package-info.java

```diff
@@ -0,0 +1,3 @@
+
+@org.jspecify.annotations.NullMarked
```

Comment:
Javadoc is missed in this file.
That gives a nice summary in the list of packages: https://docs.spring.io/spring-integration/docs/7.0.1/api/